### DISCUSSION TITLE
Allow marked tiles to show up on the world map.

### DISCUSSION THREAD

**Discussion by Tuzzz94** [runelite/runelite#15437] [Ideas]
**What's the suggestion?**

First off, in order to prevent confusion I would like to clarify that even though in this post I will be bringing up the subject of 'Tileman mode', I am actually not referring to marking 'Tileman tiles' within this post but to marking regular tiles, which are related to the 'ground marker' plugin, as the former is already a feature of the Tileman mode plugin; Which also already brings me to the conclusion that it is already technically possible to show marked tiles on the world map, so there's no objection there.

As of currently, it does not seem possible for marked tiles to show up on the world map. In the ground marker plugin, it IS possible to see marked tiles on the mini-map, but not on the world map. **Additionally**, it would also be great of tiles could be marked on the world map instead of from within the game itself.

This suggestion or request I would imagine would only apply to the ground marker plugin, but perhaps it's possible and appropriate to allocate it to a different plugin.

**Why am I requesting this?** 

I am as of currently playing a 'Tileman mode' account (I refer you to (amongst others) Settled's Tileman series on Youtube if you're not familiar with this concept)

Within this self-imposed game-mode, there are a great number of places such an account would need or want to get to whilst at the same time using as little tiles as realistically possible.

One example would be clue scroll locations. It would be great if we as tilemen could mark the tiles of places such as clue scroll loactions we eventually need to get to using 'ground markers' instead of using 'tileman tiles'. If this shows up on the world map, it allows us in a quickly see at a glance the places we would perhaps want to come by whilst making our way to another destination whilst carving out a new path. 

We'd still have to search up the initial location once before knowing where to mark a (non-tileman) tile, but it will be much more orderly and convenient to be able to mark those tiles before actually getting there, and it would be even BETTER if we could mark those tiles on the world map, so we don't actually have to physically get there in order to mark those tiles.

Note that it seems like this suggestion would only be applicable to Tileman mode, but I also think it could be potentially useful for other accounts with different purposes in mind.

Something that would also be great is if there was an option to automatically mark all the locations you could possibly need to get to for clue scrolls by tier on the world map, but this is a side-tangent because this is not part of my actual idea or suggestion in this post and therefore I may happen to request such a feature in a separate post.

Whatever may come out of this, I will thank you in advance for reading through this post and hopefully considering it :)

**Comment by Enriath** [comment]
Related issue: #14872

I still don't think the effort required to make this work would outweigh any benefits, especially when you consider that not everywhere has a world map region associated with it.
The example of marking potential unlock locations for tileman could be solved with a line in the already existing `Notes` plugin, or an external note taking program.
